book/modules/appendix3.tex

```diff
@@ -0,0 +1,172 @@
+Determinant is useful: it measures the volume distortion of a linear transformation, it helps us determine if a matrix is invertible of not, and it provides us with a simple formula for solving \(A\vec x=\vec y\) when \(\vec x\) is invertible.
+
+In Module 14, you will learn an algorithm for evaluating determinants via row reduction. However, for \(2\times 2\) and \(3\times 3\) determinants, we have methods which are more convenient than row reduction. We will explore these methods in this appendix.
```

Comment:
Add a `\label` to the start of Module 14 (right after the `\Title`) and then use a `\ref`. No absolute references allowed.

Reply:
Keep a consistent tense: "Module 14 discusses ..."

Reply:
They aren't more convenient. They are direct formulas.